mapsapi/config/base.config.js

```diff
@@ -8,21 +8,47 @@ const PUBLIC_URL = (
 
 module.exports = {
   host: '0.0.0.0',
-  user_from_host: '^([^\\.]+)\\.',
+  node_host: '0.0.0.0',
+  // user_from_host: '^([^\\.]+)\\.',
+  user_from_host: '\/user\/(.*)$',
   postgres: {
     host: 'cartodb-postgis',
   },
+  postgis: {
+    host: 'cartodb-postgis',
+  },
   redis: {
     host: 'cartodb-redis',
   },
   resources_url_templates: {
-    http: 'http://' + PUBLIC_URL + '/user/{{=it.user}}/api/v1/map',
-    https: 'https://' + PUBLIC_URL + '/user/{{=it.user}}/api/v1/map'
+    http: PUBLIC_URL + '/user/{{=it.user}}/api/v1/map',
```

Comment:
why remove the protocols?

Reply:
I was seeing request with `http://http://...` in the dev console